dev/tasks/java-jars/github.yml

```diff
@@ -164,6 +194,9 @@ jobs:
           arrow/ci/scripts/java_full_build.sh \
             $GITHUB_WORKSPACE/arrow \
             $GITHUB_WORKSPACE/arrow/java-dist
-      {{ macros.github_upload_releases(["arrow/java-dist/*.jar",
-                                        "arrow/java-dist/*.pom",
-                                        "arrow/java-dist/*.zip"])|indent }}
+      {{ macros.github_upload_releases(["arrow/java-dist/x86_64/*.jar",
+                                        "arrow/java-dist/x86_64/*.pom",
+                                        "arrow/java-dist/x86_64/*.zip",
+                                        "arrow/java-dist/aarch_64/*.jar",
+                                        "arrow/java-dist/aarch_64/*.pom",
+                                        "arrow/java-dist/aarch_64/*.zip"])|indent }}
```

Comment:
Why do we need this change?
Artifacts should be copied to `arrow/java-dist/`: https://github.com/apache/arrow/blob/master/ci/scripts/java_full_build.sh#L62-L70

Reply:
Reverted.